tests/nnstreamer_decoder/checkResult.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+
+##
+# SPDX-License-Identifier: LGPL-2.1-only
+#
+# Copyright (C) 2022 Samsung Electronics
+#
+# @file checkResult.py
+# @brief Check if the tensor decoder results is correct with the padded video.
+# @author Gichan Jang <gichan2.jang@samsung.com>
+# @date 3 Jan 2022
+# @bug No known bugs
+
+import os
+import sys
+import struct
+
+sys.path.append(os.path.dirname(os.path.abspath(os.path.dirname(__file__))))
+from test_utils import read_file
+
+def getFormatSize(colorspace):
+    if colorspace == "RGB":
+        return 3
+    elif colorspace == "BGRx":
+        return 4
+    else:
+        print ("Please check format. Your format: " + colorspace)
+        return -1
+
+
+def compareResult(golden, raw, colorspace, width, height):
```

Comment:
is it possible to move this to tests/test_utils.py?
IMO, it is better to add test-util to compare video frame.

```
##
# @brief Compare raw video frame
def compare_video_frame(file1, file2, colorspace, width, height):
...
```

Reply:
Thanks. Moved it to test_util.py